### PR TITLE
modified the exit button functionality on space-minigame screen

### PR DESCRIPTION
**Pull request by shaivikaaaa**

### CODE REVIEW

File: source/core/src/main/com/csse3200/game/components/maingame/MainGameActions.java [hunk 1]
```diff
@@ -27,5 +27,7 @@ public void create() {
   private void onExit() {
     logger.info("Exiting main game screen");
     game.setScreen(GdxGame.ScreenType.MAIN_MENU);
+
```
Comment (TDay1) [inline source/core/src/main/com/csse3200/game/components/maingame/MainGameActions.java:30]: Remove whitespace

File: source/core/src/main/com/csse3200/game/ui/terminal/MainAlertBox.java
Comment (TDay1) [inline source/core/src/main/com/csse3200/game/ui/terminal/MainAlertBox.java:1]: Add Javadocs

File: source/core/src/main/com/csse3200/game/ui/terminal/MainGameActionss.java
Comment (TDay1) [inline source/core/src/main/com/csse3200/game/ui/terminal/MainGameActionss.java:1]: Javadocs

**Review by TDay1 — APPROVED** [review]

File: source/core/src/main/com/csse3200/game/ui/terminal/MainGameActionss.java
Comment (shaivikaaaa) [reply source/core/src/main/com/csse3200/game/ui/terminal/MainGameActionss.java:1]: Yes sure.
will add it

**Pull request merged**